tests/test_subreddit.py

```diff
@@ -82,10 +81,8 @@ def test_get_subreddit_recommendations(self):
 
     @betamax()
     def test_multiple_subreddit__fetch(self):
-        with warnings.catch_warnings(record=True) as w:
-            self.r.get_subreddit('python+redditdev', fetch=True)
-            assert len(w) == 1
-            assert isinstance(w[0].message, UserWarning)
+        self.assertWarnings(UserWarning, self.r.get_subreddit,
```

Comment:
I don't understand? The assertWarnings function is a custom function that I added to PRAWTest from SO a while ago because catching them individually in each function seemed redundant.